src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -566,7 +566,7 @@ public void updateFeedbackQuestionNumber(FeedbackQuestionAttributes newQuestion)
         }
         
         adjustQuestionNumbers(oldQuestionNumber, newQuestionNumber, questions);
-        updateFeedbackQuestion(newQuestion);
+        updateFeedbackQuestionWithResponseRateCheck(newQuestion);
```

Comment:
@unyoungwax Please create a new issue for that if you haven't done so already. In fact you can go ahead and take the issue.